armi/nucDirectory/nuclideBases.py

```diff
@@ -593,18 +615,46 @@ def getMcc2Id(self):
         return self.mcc2id
 
     def getMcc3Id(self):
+        """Return the MC2-3 nuclide identification label based on the ENDF/B-VII.1 cross section library.
+
+        .. impl:: Isotopes and isomers can be queried by MC2-3 ID.
+            :id: I_ARMI_ND_ISOTOPES3
```

Comment:
The ideas for these `impl` tags, must be unique in ARMI.

Reply:
Done!